<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CAIGA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="caiga.qrc">
    <normaloff>:/icons/Oxygen/icons/caiga.png</normaloff>:/icons/Oxygen/icons/caiga.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="imageList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DropOnly</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="imageTabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>7</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="infoTab">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="origTab">
       <attribute name="title">
        <string>Orignial</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QImageViewer" name="origLabel" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="preprocessedTab">
       <attribute name="title">
        <string>Preprocessed</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QImageViewer" name="preLabel" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="processedTab">
       <attribute name="title">
        <string>Processed</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QImageViewer" name="proLabel" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="actionRevert_Project_to_Saved"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionAddImgDisk"/>
    <addaction name="actionAddImgCamera"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionExportImg_As"/>
    <addaction name="actionExportPro_As"/>
    <addaction name="actionProject_Properties"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_CAIGA"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLeftPanel_Menu"/>
    <addaction name="actionToolbar_Menu"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuImage"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionAddImgDisk_toolbar"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open An Existing Project From Disk</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent Projects</string>
   </property>
   <property name="toolTip">
    <string>Open Recent Opened Projects</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionRevert_Project_to_Saved">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Revert Project to Saved</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAddImgDisk">
   <property name="text">
    <string>Add Image From Disk</string>
   </property>
  </action>
  <action name="actionAddImgCamera">
   <property name="text">
    <string>Add Image From Camera</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout_CAIGA">
   <property name="text">
    <string>About CAIGA</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionProject_Properties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Project Properties</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo the Last Step</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo the Last Undone Step</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExportImg_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Image Information As...</string>
   </property>
  </action>
  <action name="actionAddImgDisk_toolbar">
   <property name="text">
    <string>Add Image</string>
   </property>
   <property name="toolTip">
    <string>Add Image from Disk</string>
   </property>
  </action>
  <action name="actionLeftPanel_Menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Left Panel</string>
   </property>
  </action>
  <action name="actionToolbar_Menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionExportPro_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Project Information As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QImageViewer</class>
   <extends>QWidget</extends>
   <header>qimageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="caiga.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToolbar_Menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLeftPanel_Menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
